src/main/java/com/ss/training/utopia/counter/SecurityConfiguration.java

```diff
@@ -0,0 +1,20 @@
+package com.ss.training.utopia.counter;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+/**
+ * @author Justin O'Brien
+ */
+@Configuration
+@EnableWebSecurity
+public class SecurityConfiguration extends WebSecurityConfigurerAdapter {
+
+	@Override
+	protected void configure(HttpSecurity httpSecurity) throws Exception {
+		httpSecurity.authorizeRequests().antMatchers("**").permitAll();
```

Comment:
I am guessing you did this to allow you to use that BcryptEncoder without worrying about the login page? I did this by turning off auto configuration with an annotation above the main. I think the result would be the same, but the code is a lot less.

Reply:
Nice, I'll have to remember that.